src/PasswordChecker.php

```diff
@@ -3,8 +3,7 @@
 
 class PasswordChecker
 {
-    private const MIN_LENGTH = 10; // Tied to the filtered password-blacklist.txt
```

Comment:
I assume the comment here is because the password-blacklist.txt only contains passwords of 10 chars or more. So i've prevented users from setting setMinLength less than 10.